controllers/database_secret_handler.go

```diff
@@ -0,0 +1,210 @@
+package controllers
```

Comment:
other files use 4 space intendation,

Reply:
added the same description as in the other files